README.md

```diff
@@ -1,22 +1,22 @@
 Mailgun Relayery
 ================
 
-Mailgun Relayery is a tool providing an additional security layer for the MailGun API.
+Mailgun Relayery gives you relaying with different API keys through the single-key MailGun API.
```

Comment:
actually even better: s/gives/provides? Then you can strip "you" as well:

Mailgun Relayery provides relaying ...?

Reply:
What about "Mailgun Relayery is a tool that provides relaying.."? While it looks more verbose at first glance, it has a more understandable language. "Mailgun Relayery provides relaying" is not a good start of a README, IMO.